t/05-scheduler-full.t

```diff
@@ -298,6 +299,28 @@ subtest 'Websocket server - close connection test' => sub {
       "Connection was turned off by ws server correctly";
 };
 
+
+subtest 'Worker logs correctly' => sub {
+    kill_service($wspid);
+    my $log_file = tempfile;
+    local $ENV{OPENQA_LOGFILE};
+    local $ENV{MOJO_LOG_LEVEL};
+    my $worker_pid = create_worker($k->key, $k->secret, "http://bogushost:999999", 1, $log_file);
+    my @re = qr/\[worker:debug\] Using dir .*? for host .*/,
+      qr/\[worker:info\] registering worker with .*/,
+      qr/\[worker:error\] unable to connect to host .* retry in /,
+      qr/\[worker:debug\] ## adding timer register_worker-.*/;
+    sleep(5);
+    my $i = 0;
+    for my $re (@re) {
+        my @matches = $log_file->slurp() =~ m/$re/gm;
+        ok(1 == @matches, "Worker logs correctly @{[++$i]}");
```

Comment:
+1 for using baby cart operator 😄

Reply:
You perl gurus have a name for everything!  😁

Reply:
@okurz http://search.cpan.org/dist/perlsecret/lib/perlsecret.pod#Space_station have even more fun :)